src/Symfony/Component/HttpFoundation/Session/Storage/ServiceSessionFactory.php

```diff
@@ -16,7 +16,7 @@
 /**
  * @author Jérémy Derussé <jeremy@derusse.com>
  *
- * @internal to be removed in Symfony 6
+ * @internal to be removed when bumping to symfony/framework-bundle:6.0
```

Comment:
what does this mean?

Reply:
`symfony/framework-bundle: 5.3 to 5.4.*` uses this class. 
We can delete it when `symfony/http-foundation` will conflict with `symfony/framework-bundle < 6`

Reply:
classe removed

Reply:
is FrameworkBundle 5.3 allowing symfony/http-foundation 6 ? that would be bad as it was not released yet at that time.

And to solcve that, I'd rather make FrameworkBundle 5.4 require symfony/http-foundation 5.4 only instead of allowing 6.x as well. Otherwise, once we add a conflict rule in symfony/http-foundation 6.1, FrameworkBundle 5.4 LTS would end up installing symfony/http-foundation `5.4 || 6.0` which will install 6.0 which will be unmaintained a lot before the end of the LTS.

Reply:
5.3 no, but 5.4 yes, and we should try to keep that for as long as possible, to put our own BC layer into practice
flex is the solution to stick to 5.4 for transitive deps.